client/asset/btc/btc.go

```diff
@@ -332,27 +336,19 @@ func (op *output) wireOutPoint() *wire.OutPoint {
 // auditInfo is information about a swap contract on that blockchain.
 type auditInfo struct {
 	output     *output
-	recipient  btcutil.Address
+	recipient  btcutil.Address // caution: use stringAddr, not the Stringer
 	contract   []byte
 	secretHash []byte
 	expiration time.Time
 }
 
-// Recipient returns a base58 string for the contract's receiving address. Part
-// of the asset.AuditInfo interface.
-func (ci *auditInfo) Recipient() string {
-	return ci.recipient.String()
-}
-
 // Expiration returns the expiration time of the contract, which is the earliest
-// time that a refund can be issued for an un-redeemed contract. Part of the
-// asset.AuditInfo interface.
```

Comment:
At one point there was an `asset.AuditInfo` **interface**, but it it became a concrete type at one point and these remained.